types/react-gauge-chart/react-gauge-chart-tests.tsx

```diff
@@ -1,4 +1,4 @@
 import * as React from 'react';
 import GaugeChart from 'react-gauge-chart';
 
-const Gauge: React.ReactElement = <GaugeChart id="0" />;
+const Gauge: React.ReactElement = <GaugeChart />;
```

Comment:
```suggestion
let Gauge: React.ReactElement = <GaugeChart id="gauge-chart1" />;
Gauge = <GaugeChart />;
```
let's have both, for readme-driven-development tests, OK?

Also in our PR to correct props types in package, there is no change for `id` described as `required`,
what's the status of `id`?
For the readme-driven-development, it's required and you should modify your props.
thx!

Reply:
According to [PR](https://github.com/Martin36/react-gauge-chart/pull/37) the id is optional.

Will open a PR to update the docs.
Thanks!